core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -274,7 +274,7 @@ abstract class SparkFunSuite
   protected def withLogAppender(
       appender: AbstractAppender,
       loggerNames: Seq[String] = Seq.empty,
-      level: Option[Level] = None)(
+      level: Option[Level] = Some(Level.INFO))(
```

Comment:
Let's check if there are any related test cases that need to be specified with a different log level.